### ISSUE TITLE
Please, allow people to use projection into the Find function of MongoDB node

### ISSUE THREAD

**Issue by mickaelandrieu** [n8n-io/n8n#6975]
Because we are only able to fill the first parameter of the find function, we can't filter the results:

```js
let query = mdb
	.collection(this.getNodeParameter('collection', 0) as string)
	.find(queryParameter as Document);
```
https://github.com/n8n-io/n8n/blob/4fc69b776ccea91c2a38249dc4f4ef0b191ce374/packages/nodes-base/nodes/MongoDb/MongoDb.node.ts#L149

**Comment by Joffcom** [comment]
Hey @mickaelandrieu,

As this is a feature request can you open a request on the [support forum](https://community.n8n.io) for this and we can collect votes from the community and get the change implemented unless you wanted to open a PR with the changes.

**Comment by Joffcom** [comment]
Hey @mickaelandrieu,

Did you open a request in the end? I just want to make sure I have not missed it before I close this.

**Comment by mickaelandrieu** [comment]
I'm too lazy and I don't understand why open source contribution is not living on GitHub. You can close the issue 👍

**Comment by janober** [comment]
@mickaelandrieu The reason is that we have decided that way. Having feature requests in two different locations is simply a huge mess. Nobody will find anything anymore, hard to know how much interest there really easy, and we have more work. Is simply not very efficient. We could also have decided to have them live on GitHub but that is not what happened. If that was a good or a bad decision, I do not know but that is what we ended up with. 

Sure did close it.
